crates/wasm-mutate/src/mutators/peephole/eggsy/lang.rs

```diff
@@ -24,6 +25,10 @@ define_language! {
             This operation represent a random number, if its used, every time is should represent the same random number
         */
         "rand" = Rand,
+        /*
+            This instructions is used to define unknown operands, for example when the value can come from the join of several basic blocks in a dfg
+        */
+        "undef" = Undef,
```

Comment:
How is this different from symbol, which is also just some opaque value we can manipulate but can't peek at actual value? Also, it seems like there can only be a single `undef` in an egg term as well, since we don't have a way of differentiating them, which is pretty restricting.

Reply:
This symbol is used right now to determine if the DFG is determined, if the generated DFG written as an eterm has the undef node, then the operand for some operator comes from a different basic block.